### ISSUE TITLE
Update how 'Overview' link opens for Youth Justice Nav

### ISSUE THREAD

**Issue by macho-catt** [hackforla/website#2243]
### Overview
As a user, clicking on the 'Overview' link for the Youth Justice Nav project should open a new tab to the pdf file directly. This is made as an action item of #2211 

### Details
When clicking the 'Overview' link, each project page acts differently. Some open to a google docs page, others open to a github page that hosts the pdf file, and another downloads the pdf file locally to the machine. There needs to be a standard in how the pdf files open. Developers decided during the backend meeting that pdfs should be opened as a separate tab to the file directly, since modern browsers have the ability to view pdfs.

**Example:**
[How Youth Justice Nav's pdf file currently opens](https://github.com/hackforla/product-management/blob/master/project-one-sheets/Youth-Justice-Nav-Product-One-Sheet.pdf)
[How it should open](https://www.hackforla.org/test/Access-the-Data-One-Sheet.pdf)

### Action Items
- [x] Download a copy of Youth Justice Nav's overview pdf and put it in [/assets/pdfs](https://github.com/hackforla/website/tree/gh-pages/assets/pdfs)
- [x] Change the 'Overview' link for the [markdown file](https://github.com/hackforla/website/blob/gh-pages/_projects/youthjusticenav.md) to point to the pdf file living under /assets/pdfs
- [x] Ensure that the links in both the [project page](https://www.hackforla.org/projects/) and the project's [dedicated page](https://www.hackforla.org/projects/youthjusticenav) open the pdf file properly after the change

### Resources/Instructions
[Markdown file](https://github.com/hackforla/website/blob/gh-pages/_projects/youthjusticenav.md)
[Project Page](https://www.hackforla.org/projects/youthjusticenav)
[Location for pdf files](https://github.com/hackforla/website/tree/gh-pages/assets/pdfs)
[Jekyll Static Files](https://jekyllrb.com/docs/static-files/)

**Comment by luke-karis** [comment]
Availability for this week: Thursday after 8pm, Friday after 4pm
My estimated ETA for completing this issue: not sure, I'm a little blocked right now but I'm going to look at this with fresh eyes on Thursday and if I'm still stuck I'll reach out to the channel.

**Comment by luke-karis** [comment]
I have a blocker. I believe that I have made all the changes suggested in this ticket: downloading and copying the pdf into assets/pdfs and changing the overview link in the markdown file. 

These changes can be found on this branch in my forked website repo [Update-youth-justive-nav-overview-link-2243 
](https://github.com/luke-karis/website/tree/Update-youth-justive-nav-overview-link-2243)

The overview link on Youth Justice Nav's dedicated page works fine but the one in the project page still takes you to the GitHub pdf. I'm not sure why this isn't working especially after looking at some other similar issues and PRs that seemed to have worked with just the change I have implemented.

Any help with this would be appreciated.
